yellowbrick/features/rankd.py

```diff
@@ -174,7 +174,7 @@ def rank(self, X, algorithm=None):
 
         # Extract matrix from dataframe if necessary
         if is_dataframe(X):
-            X = X.as_matrix()
+            X = X.values
```

Comment:
Fixes the deprecation warning issued in Rank2D call.

Reply:
Ah, thanks!